unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/DeployedScenesFetcher/LandWithAccess.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class LandWithAccess
+{
+    public string id => rawData.id;
+    public LandType type => rawData.type;
+    public LandRole role => rawData.role;
+    public int size => rawData.size;
+    public string name => rawData.name;
+    public string owner => rawData.owner;
+
+    public List<DeployedScene> scenes;
+    public Vector2Int[] parcels;
+    public Vector2Int @base;
```

Comment:
Can't we rename this to `baseCoords` to avoid the nasty verbatim operator?

Reply:
renamed https://github.com/decentraland/unity-renderer/pull/397/commits/1dcf87e94bcc0287a2e5753f3b525382af74fb5f